.editorconfig

```diff
@@ -18,5 +18,8 @@ trim_trailing_whitespace = false
 [tests/{**/__snapshots__/*, tests/format/**/*}]
 trim_trailing_whitespace = false
 
+[{*,changelog_unreleased/**}.md]
```

Comment:
Can you concat with the config for `[website/blog/*.md]`?

Reply:
@sosukesuzuki I have done. It seems to still work well.

```text
PS> yarn editorconfig .\changelog_unreleased\markdown\.gitkeep
yarn run v1.22.4
$ I:\prettier\node_modules\.bin\editorconfig .\changelog_unreleased\markdown\.gitkeep
charset=utf-8
indent_style=space
indent_size=2
trim_trailing_whitespace=true
end_of_line=lf
insert_final_newline=true
tab_width=2
Done in 0.15s.

PS> yarn editorconfig .\changelog_unreleased\markdown\.gitkeep.md
yarn run v1.22.4
$ I:\prettier\node_modules\.bin\editorconfig .\changelog_unreleased\markdown\.gitkeep.md
charset=utf-8
indent_style=space
indent_size=2
trim_trailing_whitespace=false
end_of_line=lf
insert_final_newline=true
tab_width=2
Done in 0.14s.

PS> yarn editorconfig .\changelog_unreleased\BLOG_POST_INTRO_TEMPLATE.md
yarn run v1.22.4
$ I:\prettier\node_modules\.bin\editorconfig .\changelog_unreleased\BLOG_POST_INTRO_TEMPLATE.md
charset=utf-8
indent_style=space
indent_size=2
trim_trailing_whitespace=false
end_of_line=lf
insert_final_newline=true
tab_width=2
Done in 0.15s.

PS> yarn editorconfig .\changelog_unreleased\BL
yarn run v1.22.4
$ I:\prettier\node_modules\.bin\editorconfig .\changelog_unreleased\BL
charset=utf-8
indent_style=space
indent_size=2
trim_trailing_whitespace=true
end_of_line=lf
insert_final_newline=true
tab_width=2
Done in 0.15s.

PS> yarn editorconfig .\README.md
yarn run v1.22.4
$ I:\prettier\node_modules\.bin\editorconfig .\README.md
charset=utf-8
indent_style=space
indent_size=2
trim_trailing_whitespace=false
end_of_line=lf
insert_final_newline=true
tab_width=2
Done in 0.18s.
```